playground/backend/containers/scio/Dockerfile

```diff
@@ -84,4 +83,18 @@ RUN chown -R appuser:appgroup /opt/playground/backend/executable_files/ \
 # Switch to appuser
 USER appuser
 
+# Let sbt download files from Maven
+RUN mkdir -p /tmp/sbt-initialize
+WORKDIR /tmp/sbt-initialize
+RUN /opt/playground/backend/new_scio_project.sh
+WORKDIR /tmp/sbt-initialize/scio
+RUN sbt "+compile"
+WORKDIR /
+RUN rm -r /tmp/sbt-initialize
+
+# Enable mitmproxy
+ENV HTTP_PROXY="http://127.0.0.1:8081"
+ENV HTTPS_PROXY="http://127.0.0.1:8081"
+ENV SBT_OPTS="-Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication"
```

Comment:
Can you explain again why we need mitmproxy?  I read the website briefly and it looks like a networking tool.  However, aren't we achieving our networking needs using kubernetes Service and other related kinds?  Can our images and the Dockerfiles that describe them serve just to ship the binary?

Reply:
It's being used to filter out which Google Cloud Storage buckets are accessible by the examples.

However, after investigating a little bit it seems that this is not the best place to have it. We discussed this with @MakarkinSAkvelon and think that the best solution would be to have a single proxy node which would act as a reverse proxy for letting external connections to the runners and also as a single point though which runners would be able to only access the allowlisted hosts. 

This approach with a single proxy node will simplify origin policy for frontend and we will be able to get rid of [the need to generate](https://github.com/apache/beam/pull/25022) `config.g.dart` completely.